### ISSUE TITLE
RDP: Select which drives to redirect

### ISSUE THREAD

**Issue by Edwin-Edward** [mRemoteNG/mRemoteNG#317]
Hello,
First of all, my compliments for your software. It's quite difficult to find on google, but it's very complete and works nicely.

This feature request is very old, as I found it in a 2010 post in your old forum: http://forum.mremoteng.org/viewtopic.php?f=4&t=95 so maybe it has already been implemented but I couldn't find it.
The function should be quite simple: in addition to select 'Yes' or 'No' to 'Disk Drives' in 'Redirect' connection config menu, it would be nice to select which drives (windows drive letters). 
Alternatively you can "inherit" a rdp configuration files, like in SSH protocol, we "inherit" a putty configuration.
 
Thanks for your efforts!

**Comment by SeethingSage** [comment]
I agree with this completely.  I used mRemote years ago, and I just tried mRemoteNG since I use a lot of RDP connections.  Not being able to select only some drives was the first thing I noticed.  I'm a network administrator.  I have tons of network drives.  When I RDP into servers on my own network, I already have those same drives present on the host server (auto-mapped via Group Policy).  The only other drive I want to connect back to is my local C drive (\\tsclient\c).  I could turn of all drives via the connection, and then manually map back to my machine's C drive, but I don't want to have to do it each time, and I'd rather not leave it mapped persistently.  In native Remote Desktop, I can choose to bring over just that one drive, my C drive.

In mRemoteNG, turning on all drives means I then have all of the same mapped drives listed twice on the host machine.  That doesn't really affect the connection - but it seriously clutters up Windows Explorer.  However, if I RDP to a server across the WAN/Internet, I might still need my local C drive, but I _certainly_ don't want ALL of my drives to be available in that case.  If I recall correctly, that can slow down the connection a lot.

I realize the program supports other protocols, and seems to have some very nice features.  But for managing my RDP connections, this one simple non-feature is a deal breaker for me.  So I'll continue to just use native RDP for now.

**Comment by DennisT1** [comment]
FWIW I'd like to add my request for this enhancement.  If I'm connecting to a low bandwidth remote site I find myself reverting back to running RDP directly so I can limit to only one drive being shared.  I find Explorer to be quite slow when you have a lot of drives redirected and the connection is slow.

BTW - Thanks for keeping this project alive.  I've been a fan since mRemote.

**Comment by PrestigeComputing** [comment]
I would also like this feature. When using Explorer on the remote system, it takes forever to display My Computer because it has to interrogate all of my local drives. I would really only want one or two of them connected.

**Comment by boomshankerx** [comment]
Bump

**Comment by sparerd** [comment]
I've scheduled other RDP work for v1.77, so this would fit in nicely

**Comment by gerootech** [comment]
is this feature still scheduled for 1.77 ??

**Comment by sparerd** [comment]
@gerootech Yes, unfortunately most of the devs have not had much time to work on the project lately.
